bin/helpers/utils.js

```diff
@@ -1045,6 +1045,14 @@ exports.getNetworkErrorMessage = (dashboard_url) => {
   return chalk.red(message)
 }
 
+exports.setNetworkLogs = (bsConfig) => {
+  if(bsConfig.run_settings.networkLogs == 'true' || bsConfig.run_settings.networkLogs == true) {
```

Comment:
Are we adding support for `networkLogs` as a CLI arg?

Reply:
No as per product spec we will only be supporting networkLogs in browserstack.json

Reply:
https://browserstack.atlassian.net/wiki/spaces/PROD/pages/3658908037/Cypress+Network+Logs#Requirements